spec/requests/notifications_controller_spec.rb

```diff
@@ -111,6 +111,40 @@ def delete_notification(resp_code, matcher)
           expect(JSON.parse(response.body)['notifications'][0]['read']).to eq(false)
         end
 
+        it "can filter notifications by type" do
```

Comment:
There are other cases for this endpoint we should test for as well. For example, we have the `limit` filter and also the `where(user: current_user)` filter which is not covered by the tests here.